### ISSUE TITLE
Release 3.3.0 fails with invalid git command

### ISSUE THREAD

**Issue by ManfredLange** [jacob-meacham/serverless-plugin-git-variables#24]
## Expected Behavior
`serverless deploy ....` with version 3.2.0 of this plugin works as expected and has been for several months.

## Actual Behavior with Version 3.3.0
Version 3.3.0 appears to break `serverless deploy ....` with the following error message:
```
Command failed: basename `git rev-parse --show-toplevel`
'basename' is not recognized as an internal or external command,
operable program or batch file.
```

## Further Details
Executing the command `git rev-parse --show-toplevel` in the terminal works as well and produces the full current path.

The command `basename` in fact does not exist in my environment.

## Environment
- Windows 10 Professional, 64 bit, english
- nodejs version 10.14.2
- npm version 6.7.0
- git: Git for Windows version 2.21.0
- serverless version 1.40.0
- Powershell 5.1.17134.590

## Update: Workaround
To work around this issue, use version 3.2.0 of the plug-in.

**Comment by jacob-meacham** [comment]
@ManfredLange thanks for the report and sorry for the trouble! I missed the dependency on basename when I was reviewing the PR. A question though - I wouldn't expect this to break anything unless you were trying to use the new `repository` field. Is that the case?

**Comment by ManfredLange** [comment]
The variables we use in `serverless.yml` are as follows:

- `git:branch`
- `git:describe`
- `git:sha1`

We don't use any other variables.

The complete line that uses these is for a `stackTag` that we use in the deployment (with `<org>` and `<repo-name>` replaced with correct values):
```
git-info: "${git:branch} - ${git:describe} - ${git:sha1} - repo: https://github.com/<org>/<repo-name>"
```

My hunch regarding this issue: It could be caused by missing double quotes for a issuing a command. Nodejs on windows can be a bit funny with that. We often have to put double quotes around commands that in particular contain file names or paths.

**Comment by ManfredLange** [comment]
@jacob-meacham Hi, just a quick question: Do you happen to have an ETA for a release that fixes this issue?

I have never created a serverless plug-in or worked on the source of one. Do you happen to have a link to good instructions for how to set up a development environment for that? If I find time, I might try creating a PR to resolve this issue. Perhaps you could point me to the place in the codebase (filename, line number) that might cause this problem? Thank you!

**Comment by jacob-meacham** [comment]
Hi @ManfredLange sorry for the delay! I just released 3.3.1 which should fix this issue (if you're curious, you can find the PR here: https://github.com/jacob-meacham/serverless-plugin-git-variables/pull/26

**Comment by ManfredLange** [comment]
Just confirmed: This issue has been resolved with version 3.3.1. Thank you! @jacob-meacham !

Keep up the great work! This is really a very useful plugin for serverless framework.
